terraso_backend/apps/graphql/schema/schema.graphql

```diff
@@ -1527,6 +1528,96 @@ String, Boolean, Int, Float, List or Object.
 """
 scalar GenericScalar
 
+"""Soil ID algorithm queries."""
+type SoilID {
+  locationBasedSoilMatches(latitude: Float!, longitude: Float!): LocationBasedSoilMatches!
+  dataBasedSoilMatches(latitude: Float!, longitude: Float!, data: SoilIDInputData!): DataBasedSoilMatches!
+}
+
+"""A ranked group of soil matches based solely on a coordinate pair."""
+type LocationBasedSoilMatches {
+  matches: [LocationBasedSoilMatch]!
+}
+
+"""A soil match based solely on a coordinate pair."""
+type LocationBasedSoilMatch {
+  soilInfo: SoilInfo!
+  match: SoilMatch!
+}
+
+"""Provides information about soil at a particular location."""
+type SoilInfo {
+  soilSeries: SoilSeries!
+  ecologicalSite: EcologicalSite
+  landCapabilityClass: LandCapabilityClass!
+}
+
+"""Information about a soil series."""
+type SoilSeries {
+  name: String!
+  taxonomySubgroup: String!
+  description: String!
```

Comment:
Question - will these be 18n key strings, or actual copy text embedded in the dataset?

Reply:
mmmm very good point, i will file a follow-up issue about this. jon's algorithm code outputs these descriptions as english text, but i think (though am not sure) that his code fetches them from a DB column somewhere else. on the other hand, having read several of the descriptions now they seem to be code-generated very simplistically from pieces of data about the soil. so i don't think there's anything to do now, but we will definitely need to figure out a better way to navigate this when localizing